sklearn/svm/classes.py

```diff
@@ -1162,13 +1162,16 @@ class OneClassSVM(BaseLibSVM, OutlierMixin):
         The offset is the opposite of `intercept_` and is provided for
         consistency with other outlier detection algorithms.
 
+    fit_status_ : int
+        0 if correctly fitted, 1 otherwise (will raise warning)
+
     Examples
     --------
     >>> from sklearn.svm import OneClassSVM
     >>> X = [[0], [0.44], [0.45], [0.46], [1]]
     >>> clf = OneClassSVM(gamma='auto').fit(X)
     >>> clf.predict(X)
-    array([-1,  1,  1,  1, -1])
+    array([-1,  1,  1,  1, -1], dtype=int64)
```

Comment:
looks like you're running on 32 bit python. I wouldn't recommend that usually. Please undo this change as it won't pass for 64 bit python.